src/blackd/__init__.py

```diff
@@ -11,13 +10,11 @@
     from aiohttp import web
     import aiohttp_cors
 except ImportError as ie:
-    print(
+    raise ImportError(
         f"aiohttp dependency is not installed: {ie}. "
         + "Please re-install black with the '[d]' extra install "
-        + "to obtain aiohttp_cors: `pip install black[d]`",
-        file=sys.stderr,
+        + "to obtain aiohttp_cors: `pip install black[d]`"
     )
```

Comment:
Now we could raise this `from ie`

Reply:
Thanks for pointing this out.  I changed to `from None` as suggested by my colleague @timj as that leads to a more friendly output, e.g.:
```
>>> import blackd
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/u2/ki/erykoff/src/black_junk/black/src/blackd/__init__.py", line 13, in <module>
    raise ImportError(
ImportError: aiohttp dependency is not installed: No module named 'aiohttp'. Please re-install black with the '[d]' extra install to obtain aiohttp_cors: `pip install black[d]`
>>> 
```

Reply:
I agree with this. While raising from the original error would be more descriptive, it's unlikely to be useful yet is likely to confuse or scare the user. If some power user really wants to dig into the error, they can edit the source code.